train_diffute_v2.py

```
@@ -0,0 +1,498 @@
+    # Move models to device and cast to dtype
+    weight_dtype = torch.float32
+    if accelerator.mixed_precision == "fp16":
+        weight_dtype = torch.float16
+    elif accelerator.mixed_precision == "bf16":
+        weight_dtype = torch.bfloat16
+
+    vae.to(accelerator.device, dtype=weight_dtype)
+    trocr_model.to(accelerator.device, dtype=weight_dtype)
+
+    # Get VAE scale factor
+    vae_scale_factor = 2 ** (len(vae.config.block_out_channels) - 1)
```

Ruff (0.8.2)

Local variable `vae_scale_factor` is assigned to but never used

Remove assignment to unused variable `vae_scale_factor`

(F841)